etc/DockerHelper.sh

```diff
@@ -75,12 +76,19 @@ _setup() {
             context="."
             buildArgs="--build-arg compiler=${compiler}"
             buildArgs="${buildArgs} --build-arg numThreads=${numThreads}"
+            if [[ "${isLocal}" == "yes" ]]; then
+                buildArgs="${buildArgs} --build-arg LOCAL_PATH=${LOCAL_PATH}"
+            fi
             imageName="${IMAGE_NAME_OVERRIDE:-"${imageName}-${compiler}"}"
             ;;
         "dev" )
             fromImage="${FROM_IMAGE_OVERRIDE:-$osBaseImage}"
             context="etc"
-            buildArgs=""
+            if [[ "${isLocal}" == "yes" ]]; then
+                buildArgs="--build-arg PREFIX=${PREFIX}"
```

Comment:
```suggestion
                buildArgs="--build-arg INSTALLER_ARGS=-prefix=${LOCAL_PATH}"
```

Reply:
LOCAL_PATH has "/home/openroad-deps/bin" not just "/home/openroad-deps"